en/README.md

```diff
@@ -7,10 +7,28 @@ MAVLink is a very lightweight, header-only message marshalling library for micro
 
 MAVLink follows a modern hybrid publish-subscribe and point-to-point design pattern: Data streams are sent / published as **topics** while configuration sub-protocols such as the [mission protocol](protocol/mission.md) or [parameter protocol](protocol/parameter.md) are point-to-point with retransmission.
 
-Because MAVLink doesn't require any additional framing it is very well suited for applications with very limited communication bandwidth. It's reference implementation in C/C++ is highly optimized for resource-constrained systems with limited RAM and flash memory. It is field-proven and deployed in many products where it serves as interoperability interface between components of different manufacturers.
+Messages are [defined within XML files](messages/README.md), which may then be converted into appropriate source code for each of the [supported languages](#supported_languages). Each XML file defines the message set supported by a particular system, also referred to as a "dialect". The reference message set that is implemented by most ground control stations and autopilots is defined in [common.xml](messages/common.md) (most dialects build on this definition).
+
+> **Tip** Because MAVLink doesn't require any additional framing it is very well suited for applications with very limited communication bandwidth. It's reference implementation in C/C++ is highly optimized for resource-constrained systems with limited RAM and flash memory. It is field-proven and deployed in many products where it serves as interoperability interface between components of different manufacturers.
```

Comment:
`reference implementation in C is ...`

I think it's just C, right?

Reply:
Yes. There is a C++ output format, but I suspect the original text this derives from a time when the C reference might be used in C++ code. In any case, thanks, I will fix.